src/components/events/event/volunteer-list/EventVolunteerList.jsx

```diff
@@ -67,10 +99,10 @@ const EventVolunteerList = ({ name, type, eventId, setViewVolunteers }) => {
       title: 'Waiver',
       dataIndex: 'waiver',
       key: 'waiver',
-      render: () => {
+      render: waiver => {
```

Comment:
let's remove the return keyword and just return the `(...)` stuff to save a line :')